### PR TITLE
Added Helm chart for DMP APD

### PR DESCRIPTION
**Pull request by iSRIDHARRAO**

### CODE REVIEW

File: K8s-deployment/Charts/dmp-apd/example-aws-resource-values.yaml [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+  hostname: dmpApd-hostname
+  annotations: {
+    cert-manager.io/cluster-issuer: "letsencrypt-staging" # use letsencrypt-prod if proper letsencrypt certs are required
+  }
+## ApiServer resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+## @param apiServer.resources.limits The resources limits for the ApiServer containers
+## @param apiServer.resources.requests The requested resources for the ApiServer containers
+##
+apiServer:
```
Comment (pranavv0) [inline K8s-deployment/Charts/dmp-apd/example-aws-resource-values.yaml:11]: remane this to a dmpApd, and remove other for other verticles

File: K8s-deployment/Charts/dmp-apd/README.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
+[![Kubescape Status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fkubescape-dmpapd%2F&label=Kubescape)](https://jenkins.iudx.io/job/kubescape-dmpapd/lastBuild/Kubescape_20Scan_20Report_20for_20dmpapd/)
+
+## Introduction
+
+Helm Chart for IUDX dmpapd Server Deployment
+
+## Create secret files
```
Comment (karun-singh) [inline K8s-deployment/Charts/dmp-apd/README.md:7]: Please add the example secrets dir with a config template and env file.

**Review by karun-singh — CHANGES_REQUESTED** [review]

File: K8s-deployment/Charts/dmp-apd/README.md (hunk 2 above)
Comment (karun-singh) [reply K8s-deployment/Charts/dmp-apd/README.md:7]: Please remove the kubescape badge as well.

File: K8s-deployment/Charts/dmp-apd/example-secrets/secrets/.dmpapd.env [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (karun-singh) [inline K8s-deployment/Charts/dmp-apd/example-secrets/secrets/.dmpapd.env:1]:
```suggestion
DMP_APD_URL=https://<dmp-apd-url>
LOG_LEVEL=INFO
```

File: K8s-deployment/Charts/dmp-apd/example-secrets/secrets/config.json [hunk 4]
```diff
@@ -0,0 +1,122 @@
+{
+  "version": "1.0",
```
Comment (karun-singh) [inline K8s-deployment/Charts/dmp-apd/example-secrets/secrets/config.json:2]:
```suggestion
  "version": "1.0",
  "zookeepers": [
    "zookeeper.zookeeper.svc.cluster.local"
  ],
```

**Review by karun-singh — CHANGES_REQUESTED** [review]
update the env and config files with these changes.

**Review by karun-singh — APPROVED** [review]
LGTM

**Pull request merged**